src/components/references/references.js

```diff
@@ -4,7 +4,60 @@ class References extends React.Component {
   render() {
     return (
       <div>
-        <h3>References</h3>
+        <h2 className='ds-h2' id='references'>References</h2>
+
+        <h3 className='ds-h3'>General References</h3>
+        <ul>
+          <li>
+            Resource Library: <a href='https://qpp.cms.gov/about/resource-library' target='_blank' rel='noopener noreferrer'>qpp.cms.gov/about/resource-library</a>
```

Comment:
should these links be `ExternalLink` components? Or am I thinking of the dev-preview docs?

Reply:
Thats in the other repo.  when i upgrade this to react 16 ill create `ExternalLink`